### PR TITLE
T dh 776

### PR DESCRIPTION
**Pull request by Fitchburgh**

### CODE REVIEW

File: src/classes/PWUTMetadataPostInstall.cls [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+				'Scheduled',
+				'Dispatched',
+				'In Progress',
+				'On Hold',
+				'Cannot Complete',
+				'Cancelled',
+				'Completed',
+				'Assigned To Contractor',
+				'Site Checkout'};
+
```
Comment (byarsawich) [inline src/classes/PWUTMetadataPostInstall.cls]: we need Did Not Complete here as well

File: src/classes/PWUTMetadataPostInstall.cls [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+				'In Progress',
+				'On Hold',
+				'Completed',
+				'Closed',
+				'Cancelled',
+				'Cannot Complete',
+				'Scheduled',
+				'Work Created',
+				'Site Checkout'};
+
```
Comment (byarsawich) [inline src/classes/PWUTMetadataPostInstall.cls]: Will need Did Not Complete here as well.

File: src/classes/PWUTMetadataPostInstall.cls-meta.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>41.0</apiVersion>
+    <status>Active</status>
```
Comment (byarsawich) [inline src/classes/PWUTMetadataPostInstall.cls-meta.xml]: 44

File: src/classes/PackageUtils.cls [hunk 4]
```diff
@@ -17,6 +17,6 @@
  * @author Ulas Kutuk
  **/
 public class PackageUtils {
-    public static final String namespace = 'pwut_toc';
-    public static String nsp = 'pwut_toc';
+    public static final String namespace = 'pwut_bjf';
+    public static String nsp = 'pwut_bjf';
 }
```
Comment (byarsawich) [inline src/classes/PackageUtils.cls]: this needs to be changed back, re added to the git repo then make git ignore changes to the file after.
git update-index --assume-unchanged src/classes/PackageUtils.cls

File: src/classes/Test_PWUTMetadataPostInstall.cls-meta.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>41.0</apiVersion>
+    <status>Active</status>
```
Comment (byarsawich) [inline src/classes/Test_PWUTMetadataPostInstall.cls-meta.xml]: 44

Comment (Fitchburgh) [reply src/classes/Test_PWUTMetadataPostInstall.cls-meta.xml]: done

File: src/classes/PackageUtils.cls (hunk 4 above)
Comment (Fitchburgh) [reply src/classes/PackageUtils.cls]: done and done

File: src/classes/PWUTMetadataPostInstall.cls-meta.xml (hunk 3 above)
Comment (Fitchburgh) [reply src/classes/PWUTMetadataPostInstall.cls-meta.xml]: done

File: src/classes/PWUTMetadataPostInstall.cls (hunk 2 above)
Comment (Fitchburgh) [reply src/classes/PWUTMetadataPostInstall.cls]: added

File: src/classes/PWUTMetadataPostInstall.cls (hunk 1 above)
Comment (Fitchburgh) [reply src/classes/PWUTMetadataPostInstall.cls]: added

**Pull request merged**